### PR TITLE
Fixing bug with text opacity + customizing icon

### PR DESCRIPTION
**Pull request by VagnerGon**
Fixes an issue with opacity when releasing the handler, which was not reseting to 1.
Also, adding a new CSS variable allowing changing the icon

### CODE REVIEW

File: src/components/SlideUnlock.vue [hunk 1]
```diff
@@ -160,7 +160,8 @@ export default defineComponent({
     methods: {
         fadeText: debounce(function() {
             const ReversePercent = ((this.sliderWidth - this.height) * (100 / this.HandlerPosition)) / 1000 - 0.1
-            this.Text.opacity = ReversePercent
+            if (this.CanMove)
```
Comment (joseph2) [inline src/components/SlideUnlock.vue:163]: Sorry, but I can't reproduce this bug

File: src/components/SlideUnlock.vue [hunk 2]
```diff
@@ -242,6 +243,7 @@ export default defineComponent({
...
...sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo0ZDhlNWY5My05NmI0LTRlNWQtOGFjYi03ZTY4OGYyMTU2ZTYiIHhtcE1NOkRvY3VtZW50SUQ9InhtcC5kaWQ6NTEyNTVEMURGMkVFMTFFNEI5NDBCMjQ2M0ExMDQ1OUYiIHhtcE1NOkluc3RhbmNlSUQ9InhtcC5paWQ6NTEyNTVEMUNGMkVFMTFFNEI5NDBCMjQ2M0ExMDQ1OUYiIHhtcDpDcmVhdG9yVG9vbD0iQWRvYmUgUGhvdG9zaG9wIENDIDIwMTQgKE1hY2ludG9zaCkiPiA8eG1wTU06RGVyaXZlZEZyb20gc3RSZWY6aW5zdGFuY2VJRD0ieG1wLmlpZDo2MTc5NzNmZS02OTQxLTQyOTYtYTIwNi02NDI2YTNkOWU5YmUiIHN0UmVmOmRvY3VtZW50SUQ9InhtcC5kaWQ6NGQ4ZTVmOTMtOTZiNC00ZTVkLThhY2ItN2U2ODhmMjE1NmU2Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+YiRG4AAAALFJREFUeNpi/P//PwMlgImBQkA9A+bOnfsIiBOxKcInh+yCaCDuByoswaIOpxwjciACFegBqZ1AvBSIS5OTk/8TkmNEjwWgQiUgtQuIjwAxUF3yX3xyGIEIFLwHpKyAWB+I1xGSwxULIGf9A7mQkBwTlhBXAFLHgPgqEAcTkmNCU6AL9d8WII4HOvk3ITkWJAXWUMlOoGQHmsE45ViQ2KuBuASoYC4Wf+OUYxz6mQkgwAAN9mIrUReCXgAAAABJRU5ErkJggg==');
```
Comment (joseph2) [inline src/components/SlideUnlock.vue:246]: New variable for icon is awesome!

**Review by joseph2 — APPROVED** [review]
Thank you for pull request :)

File: src/components/SlideUnlock.vue (hunk 1 above)
Comment (VagnerGon) [reply src/components/SlideUnlock.vue:163]: When releasing the handler, if you try some times, the text didn't reset the opacity to 1.

Comment (joseph2) [reply src/components/SlideUnlock.vue:163]: Really! I hadn't noticed this before. Thank you.

**Pull request merged**